app/uk/gov/hmrc/timetopayproxy/models/BreathingSpace.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.timetopayproxy.models
+import java.time.LocalDate
+
+import play.api.libs.json.Json
+
+case class BreathingSpace(debtRespiteFrom: LocalDate,
+                          debtRespiteTo: LocalDate,
+                          paymentDate: LocalDate,
+                          paymentAmount: BigDecimal)
+
```

Comment:
don't understand this model.. Why does BS have a payment data?

Reply:
Hi, Regarding the spec v.5 the payload was as follows:
```"breathingSpaces": {
                      "type": "array",
                      "items": [
                        {
                          "type": "object",
                          "properties": {
                            "debtRespiteFrom": {
                              "type": "string",
                              "description": "Breathing space from date"
                            },
                            "debtRespiteTo": {
                              "type": "string",
                              "description": "Breathing space to date"
                            },
                            "paymentDate": {
                              "type": "string",
                              "descprition": ""
                            },
                            "paymentAmount": {
                              "type": "integer",
                              "descprition": ""
                            }
                          }
                        }
                      ]
```